fdbserver/networktest.actor.cpp

```diff
@@ -39,16 +77,25 @@ ACTOR Future<Void> networkTestServer() {
 	state Future<Void> logging = delay( 1.0 );
 	state double lastTime = now();
 	state int sent = 0;
+	state latency_stats latency;
+	state latency_stats::sample sample;
 
 	loop {
 		choose {
 			when( NetworkTestRequest req = waitNext( interf.test.getFuture() ) ) {
+				sample = latency.tick();
```

Comment:
There's no wait on the line below, so you don't need `sample` to be `state` if you move the declaration of it into the when() here.